routes/authentication.js

```diff
@@ -0,0 +1,26 @@
+const { Router, query } = require("express");
+const jwt = require('jsonwebtoken');
```

Comment:
We have not decided whether to use jwt for authentication yet. I'm on the fence -- I don't like the fact that the tokens can't be invalidated. So let's not require it for now.

Also, as I said above, we don't need any of these route files (except index.js) yet. We should add them as the issues are work on. But let's leave them in.

Reply:
Removed and sorry again, I got a little ahead of myself there it seems.